### PR TITLE
Microsoft Windows Event Logs Modeling Rules

### PR DESCRIPTION
**Pull request by nkanon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR is created to add modeling rules pack for windows event logs.

### CODE REVIEW

File: Packs/MicrosoftWindowsEvents/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Microsoft DHCP
+
+This pack includes Cortex XSIAM content.
+
+## Collect Events from Vendor
+
+In order to use the collector, you can use one of the following options:
+1. [XDRC (XDR Collector)](#xdrc-xdr-collector) option.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:8]:
```suggestion
- [XDRC (XDR Collector)](#xdrc-xdr-collector).
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+2. [Broker VM](#broker-vm) option.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:9]:
```suggestion
- [Broker VM](#broker-vm).
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+
+
+### XDRC (XDR Collector)
+
+To create or configure the Filebeat collector, use the information described [here](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-pro-admin/cortex-xdr-collectors/xdr-collector-datasets#id7f0fcd4d-b019-4959-a43a-40b03db8a8b2).
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:16]:
```suggestion
To create or configure the Filebeat collector, use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/XDR-Collector-Datasets).
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+You can configure the vendor and product by replacing [vendor]\_[product]\_raw with *microsoft_dhcp_raw*.
+
+As cortex XSIAM provides YAML template for Windows Security Event Logs, you can use the following steps to create a collection profile:
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:20]:
```suggestion
As cortex XSIAM provides a YAML template for Windows Security Event Logs, you can use the following steps to create a collection profile:
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+   1. In Cortex XDR, select **Settings** → **Configurations** → **XDR Collectors** → **Profiles** → **+Add Profile** → **Windows**.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:22]:
```suggestion
 1. In Cortex XDR, select **Settings** → **Configurations** → **XDR Collectors** → **Profiles** → **+Add Profile** → **Windows**.
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+   2. Select **Winlogbeat**, then click **Next**.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:23]:
```suggestion
 2. Select **Winlogbeat**, then click **Next**.
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+   3. Configure the General Information parameters:
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:24]:
```suggestion
 3. Configure the General Information parameters:
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+   - Profile Name — Specify a unique Profile Name to identify the profile. The name can contain only letters, numbers, or spaces, and must be no more than 30 characters. The name you choose will be visible from the list of profiles when you configure a policy.
+
+   - Add description here — (Optional) To provide additional context for the purpose or business reason that explains why you are creating the profile, specify a profile description.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:27]:
```suggestion
   - Add description here — (Optional) Provide additional context for the purpose or business reason that explains why you are creating the profile.
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+   4. Configure the settings for the profile selected in Step 2 - To add the "Windows Security" template, select it and click **Add**.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:29]:
```suggestion
 4. Configure the settings for the profile selected in Step 2. To add the "Windows Security" template, select it and click **Add**.
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+
+### Broker VM
+To create or configure the Broker VM, use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Configure-the-Broker-VM).
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:33]:
```suggestion
To create or configure the Broker VM, use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Configure-the-Broker-VM).
```

**Comment by ShirleyDenkberg** [comment]
@nkanon Doc review completed.

**Comment by nkanon** [comment]
> @nkanon Doc review completed.

Hi @ShirleyDenkberg, thanks a lot. 
I've requested an additional review as I've updated the README for both Microsoft Event Logs and Microsoft ADFS in this pack to create differentiation.

File: Packs/MicrosoftADFS/ReleaseNotes/1_0_11.md [hunk 11]
```diff
@@ -0,0 +1,7 @@
+#### Modeling Rules
+##### Microsoft ADFS Collection
+- Updated Modeling Rules.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftADFS/ReleaseNotes/1_0_11.md:3]:
```suggestion
Updated Modeling Rules.
```

File: Packs/MicrosoftADFS/ReleaseNotes/1_0_11.md [hunk 12]
```diff
@@ -0,0 +1,7 @@
...
+
+#### Parsing Rules
+##### New: Microsoft ADFS Collection Parsing Rule
+-  Adding parsing rules for _time mapping.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftADFS/ReleaseNotes/1_0_11.md:7]:
```suggestion
Added parsing rules for _time mapping.
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
+# Microsoft Event Logs
+
+This pack includes Cortex XSIAM content.
+
+Note: The logs will be stored in the dataset named *microsoft_windows_raw*.
+
+To view logs only from the windows event log, please apply the following filter to the datamodel query: *| filter xdm.event.original_event_type="Microsoft-Windows-\*"*
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:7]:
```suggestion
To view logs only from the Windows event log, apply the following filter to the datamodel query: *| filter xdm.event.original_event_type="Microsoft-Windows-\*"*
```

File: Packs/MicrosoftWindowsEvents/README.md [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+
+
+## Collect Events from Vendor
+
+In order to use the collector, you can use one of the following options:
+- [Microsoft Event Logs](#microsoft-event-logs)
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/README.md:13]: not sure why this line and the next one is here.  They are not options for using the collector.
I would only list XDRC (XDR Collector) and Broker VM.

**Comment by ShirleyDenkberg** [comment]
@nkanon I reviewed the files again.  See my comments.

File: Packs/MicrosoftWindowsEvents/README.md (hunk 14 above)
Comment (nkanon) [reply Packs/MicrosoftWindowsEvents/README.md]: I agree, removed this extra line

**Pull request closed without merging**